lua/astrocommunity/completion/tabby-nvim/init.lua

```diff
@@ -0,0 +1,8 @@
+vim.g.tabby_keybinding_accept = "<Tab>"
+vim.g.tabby_keybinding_trigger_or_dismiss = "<C-\\>"
+
+vim.g.tabby_node_binary = "/path/to/node"
```

Comment:
This should be in the init function of vim-tabby

Reply:
`vim.g.tabby_node_binary = "/path/to/node"`

does it makes sense to add it?

Reply:
Ah good catch Luxus. 
We could just assume the current node in the path variable, i.e use the vim api to see of the tool is in the path.